assets/differentialabundance_report.Rmd

```diff
@@ -437,6 +437,11 @@ Comparisons were made between `r params$observations_type` groups defined using
 ```{r, echo=FALSE, results='asis'}
 contrasts_to_print <- contrasts
 colnames(contrasts_to_print) <- prettifyVariablename(colnames(contrasts_to_print))
+
+# If DESeq2 was run instead of Limma, add designs to report
+if (params$study_type %in% c('rnaseq')) {
+  contrasts_to_print[["DESeq2 models"]] <- paste('~ 0 + ', ifelse(contrasts_to_print$Blocking != '', make.names(contrasts_to_print$Blocking), ''), '+ ', make.names(contrasts_to_print$Variable))
```

Comment:
I'm not a fan of this, since it will require careful attention if the DESeq2 module is updated.

If you really want this, suggest you have DESeq2 (and, ideally, Limma) modules output the models in some way, so we can retrieve it here.

Reply:
Yeah, I also considered that...Alright, will for now have a look at adding some DESeq2 output